spec/configuration_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe ActionTracker::Configuration do
+  describe '#track_events flag' do
+    it 'default value is true' do
+      expect(ActionTracker::Configuration.new.track_events).to eq(true)
```

Comment:
This is the unitary test of the module, witch tests its basic behaviour. So, on this comment, you are proposing a new integration test with a initial configuration, to check if the values are correctly initalized. Am I right?